### PR TITLE
Highlight pasted input

### PR DESCRIPTION
**Pull request by ivanross**
With this PR, when you paste something, the input pasted is highlighted. 

This only works whit pasted input that has more than one char.

### CODE REVIEW

**Comment by vadimdemedes** [comment]
Hm, that's a bit unusual. What's it for?

**Comment by ivanross** [comment]
I tried to simulate the common behaviour that occurs after pasting in terminal:

gif 1

This is the result with this PR:

gif 2

I thought this would be nice to have, but maybe there should be a prop to enable this?

File: src/index.js [hunk 1]
```diff
@@ -49,7 +50,7 @@ class TextInput extends PureComponent {
 
 			let i = 0;
 			for (const char of value) {
-				if (i++ === cursorOffset) {
+				if (i >= cursorOffset - cursorWidth & i++ <= cursorOffset) {
```
Comment (vadimdemedes) [inline src/index.js]: Could you add a comment with an explanation what and how is being calculated here? What's `&` for?

**Comment by vadimdemedes** [comment]
Hm, never noticed it myself, you're right! Let's indeed expose it via a prop, disabled by default, since it would be a breaking change otherwise. We can reconsider the default value for the next major release ;)

File: src/index.js (hunk 1 above)
Comment (ivanross) [reply src/index.js]: Sorry this is a mistake

**Comment by ivanross** [comment]
Done, now there is a prop `hightlightPasted`. Default is `false` 💪

**Comment by vadimdemedes** [comment]
Could you rename it to `highlightPastedText` and document it in readme?

**Comment by ivanross** [comment]
Done!!

**Comment by vadimdemedes** [comment]
Thank you, great addition!

**Pull request merged**